mods/ra/rules/aircraft.yaml

```diff
@@ -115,13 +115,13 @@ MIG:
 		Speed: 223
 		RepulsionSpeed: 40
 		MaximumPitch: 56
+		ActionsOnIdle: ReturnToBase
```

Comment:
🙄

Reply:
No need to 🙄, it has `ReturnOnIdle` on bleed, afterall.

Reply:
The 🙄 was that the complains can be fixed simply with changing these to FlyCircle if I'm reading this right. And it was aimed more towards @abcdefg30.

Reply:
Were it so easy...